.env.example

```diff
@@ -18,9 +18,14 @@ BEE_ENDPOINT=
 CHAIN_ID=7011893055
 
 GOOGLE_STORAGE_BUCKET_NAME=blobscan-test-bucket
-GOOGLE_STORAGE_PROJECT_ID=blobscan-379218
+GOOGLE_STORAGE_PROJECT_ID=blobscan-test-project
+GOOGLE_STORAGE_API_ENDPOINT=http://localhost:4443
 GOOGLE_SERVICE_KEY=
 
+GOOGLE_STORAGE_ENABLED=false
+PRISMA_STORAGE_ENABLED=false
```

Comment:
Honestly I don't like the name :sweat_smile:. We know what prisma is because we are developing blobscan but probably someone that just wants to run it locally or deploy it to a kubernetes cluster is not familiar with Node.js ORMs and would need to look for what Prisma is. Also, what if we migrate to a different ORM in the future? It would make sense to rename this variable, right?

So far we are using Prisma as an ORM for Postgres databases. It probably works with prisma+mysql with small tweaks but we haven't tested it. For these reasons, I am inclined to think that `POSTGRES_STORAGE_ENABLED` is a better naming and more robust choice.

Lastly, let's use this as the default storage enabled, otherwise the blobs would not be stored anywhere.

In summary: `POSTGRES_STORAGE_ENABLED=true`

Reply:
I agree. Let's change it